hat/assets/js/apps/Iaso/constants/routes.js

```diff
@@ -815,4 +815,11 @@ export const routeConfigs = [
     entitySubmissionDetailPath,
     storagesPath,
     storageDetailPath,
-];
+].map(conf => {
+    const newConf = { ...conf };
+    newConf.params.push({
+        isRequired: false,
+        key: 'accountId',
+    });
+    return newConf;
+});
```

Comment:
It would save us a loop through the conf to define it directly in each conf, the way we do with `paginationParams`.